### DISCUSSION TITLE
Announcement: Project Status and Future Developments

### DISCUSSION THREAD

**Discussion by EPMatt** [EPMatt/awesome-ha-blueprints#371] [Polls]
Hi folks! I've some exciting news to share with you here.

After a long break, I'm happy to come back to the Open Source Community and pick up my work where I left it.
I have to admit that I was truly surprised to see that the project have gained so much popularity, even if not being actively maintained. **300 stars on GitHub and 10k estimated blueprint downloads on the HA Forums** are staggering numbers that show how Awesome HA Blueprints is helping so many people in automating their smart homes, faster and easier. We're truly building a great community here!

I'd like to sincerely express my gratitude to all users and contributors of the project, who provided valuable feedback and shared the blueprints with the community. This wouldn't have been possible without your help, thank you! You're amazing. 🚀 

These achievements helped me understand the impact that this project is having on the Home Assistant community, and motivated me  to put increased efforts in Awesome HA Blueprints. 
I took some time to think about the future of the project, and I've come up with a few practical goals for the upcoming months. Such activities will support the core value of Awesome HA Blueprints, which will always be the guiding light here: **building a curated collection of blueprints for Home Assistant, reliable, customizable, and fully tested by the community**.

Here are the main **project goals**:

1. **Fix and improve currently available blueprints** by processing issues and pull requests contributors and users opened during the last few months, and updating blueprints to leverage the newest features introduced in Home Assistant for templating and automations.
2. **Make it easier to contribute to the project** by simplifying the contribution process, documenting it and putting in place automations in order for new contributors to quickly evaluate their submissions.
3. **Provide better, more accessible ways to use blueprints**, by simplifying the way people search, download and interact with blueprints included in the project. This will include improvements to existing documentation, but might also require to engineer new solutions to interact with the project. The main goal is to reach more people with our work, with the ambition of **making Awesome HA Blueprints a reference point for high quality, community maintained blueprints**.

I'd love to hear your feedback about my vision for the project, so here's a poll I kindly invite you to answer to and share with fellow Home Assistant users. Your responses will help me understand what's most meaningful to you, so I'll be able to focus my time and efforts towards the goals the community finds most important.

Beside of that, I'll slowly pick up open tickets and answer your questions one by one, but due to my limited time I'll have to prioritize based on the number of affected users and impact of each issue. Thank you for your patience.

As always, I'm truly happy to collaborate with you. Any feedback, contribution or help is highly appreciated. I'm so excited for the future of Awesome HA Blueprints, and curious to see what you'd love to build together!

Thank you for your time, happy automating! 🔥

**Comment by lsismeiro** [comment]
Hi EPMatt, I'm taking this opportunity to thank you for the Awesome HA Blueprints and the way they simplify automation on Home Assistant for me and everybody that uses them. Glad to know that you will be back to take care of this wonderful work that you started. I already voted in my preferred priority. Regards, Luis Sismeiro

**Comment by jantoney** [comment]
Hey @EPMatt, Thanks for your great work! I personally voted for fixing the current automations because I feel like this PR (https://github.com/EPMatt/awesome-ha-blueprints/pull/314) has gotten a little out of control and smaller merges to fix 1 issue, then another PR for another might have been a better approach.

I downloaded your blueprints today for the first time. It was only because I'm capable of looking at GitHub issues and updating files in HA that I found and fixed my issues from the info in that PR, but I bet there are others out there that wouldn't have a clue what to do or where to start looking for a solution.

Smaller updates instead of trying to fix everything all at once would be great.

Please accept my, hopefully, constructive feedback. It's certainly not a reflection on the work you've done!
Keep it up - looking forward to future efforts.

**Comment by EPMatt** [comment]
2025 update: We are back! 🚀  Read the full post and share your updated feedback [here](https://github.com/EPMatt/awesome-ha-blueprints/discussions/746).

Thank you all for sharing your feedback here. I'm now closing this as outdated discussion, we will soon spin up a new poll for getting updated community suggestions and ideas.
Thank you all for your patience!
